.github/dependabot.yml

```diff
@@ -5,6 +5,12 @@ updates:
   schedule:
     interval: monthly
   open-pull-requests-limit: 2
+
+- package-ecosystem: nuget
```

Comment:
@jamescrosswell I'm not too familiar with dependabot but that should do the trick?

Reply:
@bitsandfoxes yes. I'm unsure about the number of PRs that can be opened. Two sounds very constraining for a weekly period.

Reply:
On the other hand not very busy/distracting, we can adjust as we go